cli: Ignore failure when reading /etc/nmstate folder

cathay4t opened the pull request:
When /etc/nmstate not exists or other failure when reading it, we should
ignore it and exist 0.

Integration test case included.

cgwalters commented on rust/src/cli/service.rs:
```diff
@@ -15,13 +15,23 @@ pub(crate) fn ncl_service(
         .unwrap_or(crate::DEFAULT_SERVICE_FOLDER);
 
-    let config_files = get_config_files(folder)?;
+    let config_files = match get_config_files(folder) {
```
You might consider using https://crates.io/crates/liboverdrop

cathay4t committed:
cli: Ignore failure when reading /etc/nmstate folder

When /etc/nmstate not exists or other failure when reading it, we should
ignore it and exist 0.

Integration test case included.

Signed-off-by: Gris Ge <fge@redhat.com>
